modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployService.java

```diff
@@ -566,4 +571,30 @@ private void maskSensitiveFields(DeployServiceEntity deployServiceEntity) {
             }
         }
     }
+
+    private JsonObjectSchema convertJsonObjectSchemaEntityToJsonObjectSchema(JsonObjectSchemaEntity entity) {
```

Comment:
we just need to generate the JSON schema object as a JSON string and store in DB.